airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_source.py

```diff
@@ -32,3 +33,6 @@ def check_connection(self, logger: logging.Logger, config: Mapping[str, Any]) ->
           The error object will be cast to string to display the problem to the user.
         """
         return self.connection_checker.check_connection(self, logger, config)
+
+    def all_streams(self, config: Mapping[str, Any]) -> List[Stream]:
+        return self.streams(config=config)
```

Comment:
For non concurrent low-code sources, these are equivalent, but we override this implementation in `ConcurrentDeclarativeSource` to create a single list of both concurrent and synchronous sources so that we properly generate catalogs and other things